ocaml/xapi-idl/storage/storage_skeleton.ml

```diff
@@ -27,193 +25,193 @@ module UPDATES = struct
 
-  let attach3 ctx ~dbg ~dp ~sr ~vdi ~vm ~read_write = u "VDI.attach3"
+  let attach3 ctx ~dbg ~dp ~sr ~vdi ~vm ~read_write = unimplemented __FUNCTION__
 
-  let activate ctx ~dbg ~dp ~sr ~vdi = u "VDI.activate"
+  let activate ctx ~dbg ~dp ~sr ~vdi = unimplemented __FUNCTION__
 
-  let activate3 ctx ~dbg ~dp ~sr ~vdi ~vm = u "VDI.activate3"
+  let activate3 ctx ~dbg ~dp ~sr ~vdi ~vm = unimplemented __FUNCTION__
 
-  let activate_readonly ctx ~dbg ~dp ~sr ~vdi ~vm = u "VDI.activate_readonly"
+  let activate_readonly ctx ~dbg ~dp ~sr ~vdi ~vm = unimplemented __FUNCTION__
 
-  let deactivate ctx ~dbg ~dp ~sr ~vdi ~vm = u "VDI.deactivate"
+  let deactivate ctx ~dbg ~dp ~sr ~vdi ~vm = unimplemented __FUNCTION__
 
-  let detach ctx ~dbg ~dp ~sr ~vdi ~vm = u "VDI.detach"
+  let detach ctx ~dbg ~dp ~sr ~vdi ~vm = unimplemented __FUNCTION__
 
   let epoch_end ctx ~dbg ~sr ~vdi ~vm = ()
 
-  let get_url ctx ~dbg ~sr ~vdi = u "VDI.get_url"
+  let get_url ctx ~dbg ~sr ~vdi = unimplemented __FUNCTION__
 
-  let similar_content ctx ~dbg ~sr ~vdi = u "VDI.similar_content"
+  let similar_content ctx ~dbg ~sr ~vdi = unimplemented __FUNCTION__
 
-  let get_by_name ctx ~dbg ~sr ~name = u "VDI.get_by_name"
+  let get_by_name ctx ~dbg ~sr ~name = unimplemented __FUNCTION__
 
-  let set_content_id ctx ~dbg ~sr ~vdi ~content_id = u "VDI.set_content_id"
+  let set_content_id ctx ~dbg ~sr ~vdi ~content_id = unimplemented __FUNCTION__
 
-  let compose ctx ~dbg ~sr ~vdi1 ~vdi2 = u "VDI.compose"
+  let compose ctx ~dbg ~sr ~vdi1 ~vdi2 = unimplemented __FUNCTION__
 
-  let add_to_sm_config ctx ~dbg ~sr ~vdi ~key ~value = u "VDI.add_to_sm_config"
+  let add_to_sm_config ctx ~dbg ~sr ~vdi ~key ~value = unimplemented __FUNCTION__
 
   let remove_from_sm_config ctx ~dbg ~sr ~vdi ~key =
-    u "VDI.remove_from_sm_config"
+    unimplemented __FUNCTION__
 
-  let enable_cbt ctx ~dbg ~sr ~vdi = u "VDI.enable_cbt"
+  let enable_cbt ctx ~dbg ~sr ~vdi = unimplemented __FUNCTION__
 
-  let disable_cbt ctx ~dbg ~sr ~vdi = u "VDI.disable_cbt"
+  let disable_cbt ctx ~dbg ~sr ~vdi = unimplemented __FUNCTION__
 
-  let data_destroy ctx ~dbg ~sr ~vdi = u "VDI.data_destroy"
+  let data_destroy ctx ~dbg ~sr ~vdi = unimplemented __FUNCTION__
 
   let list_changed_blocks ctx ~dbg ~sr ~vdi_from ~vdi_to =
-    u "VDI.list_changed_blocks"
+    unimplemented __FUNCTION__
 end
 
-let get_by_name ctx ~dbg ~name = u "get_by_name"
+let get_by_name ctx ~dbg ~name = unimplemented __FUNCTION__
 
 module DATA = struct
-  let copy ctx ~dbg ~sr ~vdi ~vm ~url ~dest = u "DATA.copy"
+  let copy ctx ~dbg ~sr ~vdi ~vm ~url ~dest = unimplemented __FUNCTION__
 
-  let mirror ctx ~dbg ~sr ~vdi ~vm ~dest = u "DATA.mirror"
+  let mirror ctx ~dbg ~sr ~vdi ~vm ~dest = unimplemented __FUNCTION__
 
-  let stat ctx ~dbg ~sr ~vdi ~vm ~key = u "DATA.stat"
+  let stat ctx ~dbg ~sr ~vdi ~vm ~key = unimplemented __FUNCTION__
 
   let import_activate ctx ~dbg ~dp ~sr ~vdi ~vm =
-    u "DATA.MIRROR.import_activate"
```

Comment:
This was moved outside of DATA.MIRROR but the name wasn't changed, so changing to __FUNCTION__ is already helping!